mmv1/third_party/tgc/tests/data/example_compute_url_map.json

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "name": "//compute.googleapis.com/projects/{{.Provider.project}}/global/urlMaps/urlmap",
+        "ancestry_path": "{{.Ancestry}}/project/{{.Provider.project}}",
+        "asset_type": "compute.googleapis.com/UrlMap",
+        "resource": {
+            "version": "beta",
+            "discovery_document_uri": "https://www.googleapis.com/discovery/v1/apis/compute/beta/rest",
```

Comment:
Can you check if you can update the autogenerated code to return "v1/rest" as the suffix instead of "beta/rest"

Reply:
Please ignore this for now, as discussed offline. We can figure out later how to resolve this specific issue.